modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-channel-test/src/testIntegration/java/com/liferay/headless/commerce/admin/channel/resource/v1_0/test/ChannelResourceTest.java

```diff
@@ -290,6 +274,271 @@ private long _getCommerceChannelRelId() throws Exception {
 		return _commerceChannelRel.getCommerceChannelRelId();
 	}
 
+	private void _testPatchChannelByExternalReferenceCodeWithAccountExternalReferenceCode()
+		throws Exception {
+
+		Channel postChannel =
```

Comment:
is this the right name? you are calling a method that is not the post

Reply:
it's called like this in the base class
```
@Test
	public void testPatchChannelByExternalReferenceCode() throws Exception {
		Channel postChannel =
			testPatchChannelByExternalReferenceCode_addChannel();

		Channel randomPatchChannel = randomPatchChannel();
....
```